circle.yml

```diff
@@ -51,7 +51,8 @@ dependencies:
   override:
     - .circle/docker-compose.sh pull ${DISTRO}
   post:
-    - .circle/docker-compose.sh build ${DISTRO}
+    - .circle/docker-compose.sh build_st2 ${DISTRO}
```

Comment:
This approach (2 commands) removes parallelism and brings additional overhead.
FYI `st2mistral` and `st2` packages are built in parallel and tested by Rspec tests.


You can just pass 1 env variable `ST2_PACKGES`: 

- `ST2_PACKAGES=st2mistral st2` for `st2-packages` repo
- `ST2_PACKAGES=st2` for `st2` repo
- `ST2_PACKAGES=st2mistral` for `mistral` repo

and avoid 2 commands: `build_st2` and `build_st2mistral`